static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -118,17 +119,6 @@ function Task({task, completed, hidePanel}: TaskProps) {
     [task, organization, router, hidePanel]
   );
 
-  const handleMarkComplete = useCallback(
-    (taskKey: OnboardingTaskKey) => {
-      updateOnboardingTask(api, organization, {
-        task: taskKey,
-        status: 'complete',
-        completionSeen: true,
-      });
-    },
-    [api, organization]
-  );
```

Comment:
we no longer fetch the project in the new experience, so we can remove this function